istio/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "manifest_version": "0.1.1",
```

Comment:
Can we bump to `1.0.0` and remove `max_agent_version` and `min_agent_version`?

Reply:
Yep